ROOTFS/etc/waggle/sanity/fatal/static_psu_control.test

```diff
@@ -0,0 +1,159 @@
+#!/bin/bash
+
+# Written by Kenan with plenty of Joe's code mixed in
+
+# A static test to make sure all of the PSU stuff works
+
+# the other interactive tests have this 'Begin' line so I'll also include it
+
+pf="Static PSU test"
+echo "$pf: Begin"
+
+acok=$(waggle-get-config -s wagman -k gpio-acok3)
+pg1=$(waggle-get-config -s wagman -k gpio-pg1)
+pg2=$(waggle-get-config -s wagman -k gpio-pg2)
+pg3=$(waggle-get-config -s wagman -k gpio-pg3)
+pg4=$(waggle-get-config -s wagman -k gpio-pg4)
+
+gpiopath="/sys/class/gpio"
+
+
+# not my code, copied from Joe's PSU test
+gpio_high() { 
+
+        gpio=$1
+        if [ ! -f $gpiopath/gpio$gpio/value ]; then 
+                echo "$pf: Cannot find value of device $gpio"
+                exit 1
+        fi
+
+        if [ $(cat $gpiopath/gpio$gpio/value) -eq 1 ]; then 
+                return 0
+        else 
+                return 1
+        fi
+}
+
+# Not my code, taken from Joe's PSU test
+gpio_cleanup()
+{
+        if [ -d $gpiopath/gpio$acok ]; then
+                echo $acok > $gpiopath/unexport
+        fi
+        if [ -d $gpiopath/gpio$pg1 ]; then
+                echo $pg1 > $gpiopath/unexport
+        fi
+        if [ -d $gpiopath/gpio$pg2 ]; then
+                echo $pg2 > $gpiopath/unexport
+        fi
+        if [ -d $gpiopath/gpio$pg3 ]; then
+                echo $pg3 > $gpiopath/unexport
+        fi
+        if [ -d $gpiopath/gpio$pg4 ]; then
+                echo $pg4 > $gpiopath/unexport
+        fi
+}
+trap gpio_cleanup EXIT
+
+
+# Not my code, taken from Joe's PSU test
+
+# export the GPIOs and set to input
+if [ ! -d $gpiopath/gpio$acok ]; then
+        echo $acok > $gpiopath/export
```

Comment:
It would be nice to know if this export fails and return an appropriate error code.

These GPIOs are only available because of a chip on the Wagman hardware.  If that wagman hardware is not working properly the GPIOs will not be enumerated and this export could fail.  We should try to catch that use-case.

In fact, maybe this "export if not already exported" code would be a function...